--- beesu-2.7/nautilus-beesu-manager-1.7/libexec/scripts/Open Nautilus Here.nfix	2012-03-01 17:02:55.344922134 -0500
+++ beesu-2.7/nautilus-beesu-manager-1.7/libexec/scripts/Open Nautilus Here	2012-03-01 17:03:00.214873732 -0500
@@ -6,6 +6,6 @@ else
      while [ ! -z "$1" -a ! -d "$base/$1" ]; do shift; done
      dir="$base/$1"
 fi
-beesu - nautilus --browser "$dir"
+beesu - nautilus "$dir"
 # F12
 /bin/true
